ts-model-api/src/ChildrenAccessor.ts

```diff
@@ -21,34 +21,55 @@ export abstract class ChildrenAccessor<ChildT extends ITypedNode> implements Ite
 }
 
 export class ChildListAccessor<ChildT extends ITypedNode> extends ChildrenAccessor<ChildT> {
-  constructor(parentNode: INodeJS, role: string | undefined) {
-    super(parentNode, role);
+  constructor(parentNode: INodeJS, role: string | undefined, concept?: IConceptJS | undefined) {
```

Comment:
You can just use `val conceptRef = concept?.getUID()?.let { ConceptReference(it) } ?: role?.let { node.concept?.getChildLink(it) }?.targetConcept` in `NodeAdapterJS.addNewChild` instead.

Reply:
Wow that really seems much simpler. I am gonna adress this in https://github.com/drik98/modelix.core/pull/2